cassandra_template.yaml

```diff
@@ -338,7 +338,7 @@ index_summary_resize_interval_in_minutes: 60
 # buffers. Enable this to avoid sudden dirty buffer flushing from
 # impacting read latencies. Almost always a good idea on SSDs; not
 # necessarily on platters.
-trickle_fsync: false
+trickle_fsync: true
```

Comment:
Because we are using SSDs anyway, and not having fsync enabled makes me a bit nervous.